rllib/core/models/tests/test_catalog.py

```diff
@@ -256,6 +263,87 @@ def test_get_dist_cls_from_action_space(self):
                 actions = dist.sample()
                 assert action_space.contains(actions.numpy()[0])
 
+    def test_catalog_and_algorithm_config(self):
+        """Test if we can pass catalog to algorithm config and it ends up inside
+        RLModule and is used to build models there."""
+
+        class MyCatalog(PPOCatalog):
+            pass
+
+        config = (
+            PPOConfig()
+            .rl_module(rl_module_spec=SingleAgentRLModuleSpec(catalog_class=MyCatalog))
+            .framework("torch")
+        )
+
+        algo = config.build(env="CartPole-v0")
+        assert algo.get_policy("default_policy").model.config.catalog_class == MyCatalog
```

Comment:
use the self.AssertXX stuff for these types of checks within Unittest classes.

Reply:
I would instead check if config.get_catalog() returns a catalog of this instance.

Reply:
Yep, my bad 🙂